### PR TITLE
readd overlap in the sampling

### PR DESCRIPTION
**Pull request by coolo**
This just makes the image smoother - and less victim of bad time aligning

### CODE REVIEW

File: snd2png/snd2png.cpp [hunk 1]
```diff
@@ -66,6 +67,11 @@ max_row_value (double *points, int count, int index)
 int
 main (int argc, char *argv[])
 {
+  if (argc < 3) {
+    fprintf (stderr, "Usage: snd2png [soundfile] [imagefile]\n");
```
Comment (aaannz) [inline snd2png/snd2png.cpp]: Doesn't `[]` usually mark optional argument?

Comment (coolo) [inline snd2png/snd2png.cpp]: that has true in it ;)

**Comment by okurz** [comment]
-1 for typo in commit message, otherwise considered +1

**Comment by coolo** [comment]
fixed the argument description

**Pull request merged**